vcf2fhir/test/test_unit.py

```diff
@@ -6,96 +6,98 @@
 suite = doctest.DocTestSuite(vcf2fhir)
 
 
+def _test_utility_function(
```

Comment:
Why are we creating this method ? what existed before was good. Please revert this too.
Unrelated but the method name or signature does not convey anything about the method itself.